provisioning/meta/playbooks/transactor-setup.yml

```diff
@@ -0,0 +1,20 @@
+---
+- hosts: transactor-provisioning
+  tasks:
+  - name: update system
+    apt: upgrade=yes
+  - name: setup git ppa repo
+    apt_repository: repo=ppa:git-core/ppa
+  - name: install git
```

Comment:
It's very old -- 1.9.x something.
I also were getting some 404s for the file in repo in earlier instance setups, out of which this playbook evolved.